app/uk/gov/hmrc/gform/gform/EmailAuthController.scala

```diff
@@ -199,14 +205,18 @@ class EmailAuthController(
 
       val (pageErrors, maybeCodeFieldError) = error match {
         case Some(true) =>
+          val errorMessage = maybeCodeLength match {
+            case Some(value) if value === 4 => "emailAuth.error.invalidCode"
```

Comment:
I think this should work as well but shorter
```
 case Some(4) => "emailAuth.error.invalidCode"
```

Reply:
Done